ykrt/src/trace/mod.rs

```diff
@@ -221,10 +221,13 @@ impl IRTrace {
     }
 }
 
-#[derive(Debug)]
+struct SendSyncPtr<T>(T);
```

Comment:
Should we make it:

```
struct SendSyncPtr<T>(*const T);
```

So that we can't accidentally put non-pointers inside?

Reply:
Good idea! Done in 9c5a16b -- notice I've also changed the struct to `SendSyncConstPtr` because this can't be used with `*mut` pointers (if we have them at some point), and we'd need a second struct for that.